lib/registrant.rb

```diff
@@ -0,0 +1,25 @@
+class Registrant
+  attr_reader :name, :age, :permit, :license_data
+  def initialize(name, age, permit = false, license_data = {:written => false, :license => false, :renewed => false} )
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = license_data
+  end
+
+  def permit?
+    @permit
+  end
+  
+  def license_data
+    @license_data
+  end
+
+  def earn_permit
+    if @age >= 16
+      @permit = true
+    else
+      @permit = false
```

Comment:
Since `@permit` defaults to `false` we only need the `if` portion here where we set it to true. If the condition is not met (`@age >= 16`) then nothing changes and it will remain false as set by the default value.